BasicSample/Assets/ARAnchor/Scripts/AnchorsSample.cs

```diff
@@ -17,11 +17,9 @@ namespace Microsoft.MixedReality.OpenXR.BasicSample
     /// again near these anchors toggles their persistence, backed by the <c>XRAnchorStore</c>.
     /// </summary>
     [RequireComponent(typeof(ARAnchorManager))]
+    [RequireComponent(typeof(ARSessionOrigin))]
     public class AnchorsSample : MonoBehaviour
     {
-        [SerializeField]
-        private GameObject m_anchorsContainer;
```

Comment:
why is this removed? what is its significance?

Reply:
Now we use ARSessionOrigin.trackablesParent now instead, so there's no need for us to have our own object to put anchors in